charts/incubator/lldap/templates/_secrets.tpl

```diff
@@ -0,0 +1,21 @@
+{{/* Define the secrets */}}
+{{- define "lldap.secrets" -}}
+
+{{- $jwtSecret := "" }}
+{{- $jwtSecret = .Values.lldap.configuration.jwt | default (randAlphaNum 48) | quote }}
```

Comment:
Not going to work, this will randomise it every run.
Needs complicated lookup code.